### ISSUE TITLE
Copy paste from a note to a Word doc or anywhere else, images are not pasted

### ISSUE THREAD

**Issue by flaviozluca** [Intermesh/groupoffice#919]
Copy paste from a note to a Word doc or anywhere else, images are not pasted

See video, please. 

Thanks.

https://user-images.githubusercontent.com/44004645/204610835-4bc43dd1-fba3-49b9-8fb6-b96b0b8804c8.mp4

**Comment by mschering** [comment]
I don't think we can do anything about it. It's word not handling that. Probably because it can't download the images unauthenticated.

**Comment by flaviozluca** [comment]
Friend,
It works fine copy x pasting images in emails for GMail and tested with notes in icewarp, also worked.
See video.
Thanks.

https://user-images.githubusercontent.com/44004645/207933341-6be78b8d-5425-46af-b534-104e27635e52.mp4

**Comment by flaviozluca** [comment]
Today I had a surprise. It worked, I copy x pasted from a note with images to our helpdesk and it worked. To Word too.

It seems NOTES that have images that are inside GO don't paste.

This note has an image with an absolute link, then it works.

Evidence attached of both notes, the one that pastes image (with video) and the one that doesn't paste image (video I sent before).

Thanks.
[doesnt paste.txt](https://github.com/Intermesh/groupoffice/files/10393864/doesnt.paste.txt)
[pastes.txt](https://github.com/Intermesh/groupoffice/files/10393866/pastes.txt)

https://user-images.githubusercontent.com/44004645/211850275-6c9847f5-2c01-46b0-bdb2-dde34ee4651c.mp4

**Comment by mschering** [comment]
Sorry this is not a GO issue.

**Comment by flaviozluca** [comment]
Friend,

Tested with icewarp mail server, it works fine, pastes embedded image. GO doesnt.

See video.

**Comment by flaviozluca** [comment]
evid

[compare.zip](https://github.com/user-attachments/files/25797762/compare.zip)

**Comment by mschering** [comment]
Ok, I've used a a different approach of generating img url's and now it will work.

**Comment by flaviozluca** [comment]
Thanks a lot!
